components/leaflet-maps/ClustersMap.vue

```diff
@@ -195,7 +195,53 @@
        })
 
        return optionalsWithValue
-     }
+     },
+     getFormatDate: function (date,precision){
```

Comment:
Sorry Ruby, I think I might have originally asked for this function name to be changed. After reading around, I can't really find a good way to describe what we are doing here so I think (possibly the original) the name of getJulianDate() should suffice. Sorry for messing you around with this

Reply:
thanks for changing